src/Microsoft.ML.TorchSharp/NasBert/NasBertTrainer.cs

```diff
@@ -583,13 +584,15 @@ private IList<int> PrepInputTokens(ref ReadOnlyMemory<char> sentence1, ref ReadO
                 getSentence1(ref sentence1);
                 if (getSentence2 == default)
                 {
-                    return new[] { 0 /* InitToken */ }.Concat(tokenizer.EncodeToConverted(sentence1.ToString())).ToList();
+                    List<int> newList = new List<int>(tokenizer.EncodeToConverted(sentence1.ToString()));
+                    newList.Insert(0, 0);
```

Comment:
>newList.Insert(0, 0); [](http://example.com/codeflow?start=20&length=21)

worth a comment telling insert InitToken.

Reply:
Just added in a comment for it.